pkgs/top-level/python-packages.nix

```diff
@@ -6732,6 +6734,8 @@ in {
 
   splinter = callPackage ../development/python-modules/splinter { };
 
+  spotify-cli-linux = callPackage ../development/python-modules/spotify-cli-linux { };
```

Comment:
This package set is for libraries primarily, a cli does not belong here and should move into the top-level

Reply:
Ok, sorry, moving it over..